<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookAddingForm</class>
 <widget class="QWidget" name="bookAddingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Book</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="genreEdit">
        <property name="placeholderText">
         <string>Genre</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="pageCountEdit">
        <property name="placeholderText">
         <string>Page count</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Author name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="authorsList">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Publisher</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="publishersList">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="countEdit">
        <property name="placeholderText">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLineEdit" name="priceEdit">
        <property name="placeholderText">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
